third-party-projects.md

```diff
@@ -9,6 +9,12 @@ navigation:
 
 This page tracks external software projects that supplement Apache Spark and add to its ecosystem.
 
+To add a project, open a pull request against the [spark-website](https://github.com/apache/spark-website) 
+repository. Add an entry to this markdown file, then run `jekyll build` to generate the HTML too. Include
```

Comment:
maybe point to the markdown file directly here instead of "this markdown file"?

Reply:
Hm, although Spark's trademark policy says things like "sparkly" aren't OK, I don't really see that in https://www.apache.org/foundation/marks/ ? It doesn't strike me as confusingly similar.

Reply:
We've received the feedback that these aren't OK on some previous trademarks@ threads, but it could be good to clarify. It's true that the policy doesn't seem to list examples, at least on that page.